nested_open_struct.gemspec

```diff
@@ -6,7 +6,10 @@ Gem::Specification.new do |s|
   s.homepage    = "http://github.com/porras/nested_open_struct"
   s.summary     = "Thin wrapper around OpenStruct to make structs 'nested'"
   s.description = "Thin wrapper around OpenStruct to make structs 'nested'"
- 
+
   s.files        = Dir.glob("lib/**/*") + %w(LICENSE README.md)
   s.require_path = 'lib'
-end
\ No newline at end of file
+
+  s.add_development_dependency 'minitest', '~> 5.8'
+  s.add_development_dependency 'rake',     '~> 10.4'
```

Comment:
> With this PR, there's still a Gemfile, a Gemfile.lock and hence a dependency on bundler.

I wouldn’t say `Gemfile` existence means there’s a dependency on Bundler – it’s just being nice to Bundler users (which are virtually all Ruby devs). As for `Gemfile.lock` – let me [quote Yehuda](http://yehudakatz.com/2010/12/16/clarifying-the-roles-of-the-gemspec-and-gemfile/):

> In general, a gem’s `Gemfile` should contain the RubyGems source and a single `gemspec` line. Do not check your `Gemfile.lock` into version control, since it enforces precision that does not exist in the `gem` command, which is used to install gems in practice. Even if the precision could be enforced, you wouldn't want it, since it would prevent people from using your library with versions of its dependencies that are different from the ones you used to develop the gem.

I think the above is a good rule of thumb (with obvious exceptions like features of `Gemfile` that can’t be expressed in a `gemspec`.)

> Would there be a way of getting rid of it (installing the development dependencies from the gemspec)?

In the future – yes, `gem install --file nested_open_struct.gemspec` should eventually work (it doesn’t at the moment because `--file` support is still very buggy and RubyGems devel is underfunded).

Currently it’s _technically_ possible – `gem build nested_open_struct.gemspec && gem install --development nested_open_struct-*.gem` – but it’s definitely much simpler to just use Bundler.